Update README with setup instructions (#7)
voltrondata-labs/2024-arrow-format-tutorial: README.md

#  A deep dive into the Arrow Columnar format with pyarrow and nanoarrow

> Repository for the Arrow Columnar Format Tutorial at PyCon DE 2024.

Reviewer: ```suggestion
Repository for the Arrow Columnar Format Tutorial at PyCon DE & PyData Berlin 2024.
```

> Apache Arrow has become a de-facto standard for efficient in-memory columnar data representation. You might have heard about Arrow or using Arrow, but do you understand the format and why it’s so useful? This tutorial will dive deep into the details of the Arrow columnar format, the different types and buffer layouts, and explore those details interactively using the pyarrow and nanoarrow libraries.

Reviewer 2: ```suggestion
Apache Arrow has become a de-facto standard for efficient in-memory columnar data representation. You might have heard about Arrow or using Arrow, but do you understand the format and why it's so useful? This tutorial will dive deep into the details of the Arrow columnar format, the different types and buffer layouts, and explore those details interactively using the pyarrow and nanoarrow libraries.
```
Not sure why it's using right single quotation mark instead of apostrophe

Author: Probably from copying this from the PyConDE website

## Setup

To run this tutorial locally, follow those steps:

**Clone this repository**

    git clone https://github.com/voltrondata-labs/2024-arrow-format-tutorial.git

**Install the necessary packages**

> 

Reviewer 2: ```suggestion

For non conda setups we recommend to create a virtual environment.

   cd 2024-arrow-format-tutorial
   python -m venv .venv
   source .venv/bin/activate

```
